### PR TITLE
SAML Single Logout

### PR DESCRIPTION
**Pull request by alexanderzobnin**
**What this PR does / why we need it**:
This is an open source part of SAML Single Logout PR.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by sakjur — APPROVED** [review]

**Comment by marefr** [comment]
@alexanderzobnin need to do a `go mod tidy` to fix failed build

File: docs/sources/enterprise/saml.md [hunk 1]
```diff
@@ -41,24 +41,26 @@ In terms of initiation:
...
+| ----------------------------------------------------------- | -------- | --------------------------------------------------------------------------------------------- | ------------- |
 | `enabled`                                                   | No  | Whether SAML authentication is allowed                                                             | `false`       |
+| `single_logout`                                             | No  | Whether SAML Single Logout enabled                                                                 | `false`       |
 | `certificate` or `certificate_path`                         | Yes | Base64-encoded string or Path for the SP X.509 certificate                                         |               |
 | `private_key` or `private_key_path`                         | Yes | Base64-encoded string or Path for the SP private key                                               |               |
+| `signature_algorithm`                                       | No  | Signature algorithm using for signing requests to the IdP. Supported values are rsa-sha1, rsa-sha256, rsa-sha512. |              |
```
Comment (marefr) [inline docs/sources/enterprise/saml.md:50]:
```suggestion
| `signature_algorithm`                                       | No  | Signature algorithm used for signing requests to the IdP. Supported values are rsa-sha1, rsa-sha256, rsa-sha512. |              |
```

**Review by marefr — APPROVED** [review]
LGTM, but added a suggestion

**Review by marefr — APPROVED** [review]
LGTM (but don't have the details of the feature)

**Review by sakjur — APPROVED** [review]

**Pull request merged**